try_it_results.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
should this file be generated on ci side?

Reply:
I guess it should be in gitignore

Reply:
Yep, I included it as a reference